internal/app/delete_data.go

```diff
@@ -0,0 +1,86 @@
+package app
+
+import (
+	"net/http"
+
+	"github.com/KznRkjp/go-keeper.git/internal/database"
+	"github.com/KznRkjp/go-keeper.git/internal/middleware/mlogger"
+	"github.com/go-chi/chi"
+)
+
+// DeleteDataLP - удаление записи logopass po ID записи и ID пользователя
+func DeleteDataLP(res http.ResponseWriter, req *http.Request) {
+	recordId := chi.URLParam(req, "id")
+	mlogger.Info("Deleting data - LP with id: " + recordId)
+	userId := checkCookie(req)
+	if userId == 0 {
+		mlogger.Info("User not found")
+		res.WriteHeader(http.StatusNotFound)
+		return
+	}
+
+	err := database.DeleteDataLP(&recordId, &userId, req.Context())
+	if err != nil {
+		mlogger.Info(err.Error())
+		res.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	res.WriteHeader(http.StatusOK)
+}
+
+func DeleteDataBC(res http.ResponseWriter, req *http.Request) {
+	recordId := chi.URLParam(req, "id")
+	mlogger.Info("Deleting data - BC with id: " + recordId)
+	userId := checkCookie(req)
+	if userId == 0 {
+		mlogger.Info("User not found")
+		res.WriteHeader(http.StatusNotFound)
+		return
+	}
+
+	err := database.DeleteDataBC(&recordId, &userId, req.Context())
+	if err != nil {
+		mlogger.Info(err.Error())
```

Comment:
К логируемым ошибка лучше добавлять их контекст, т.е. поясняющее сообщение о том, что именно пошло не так. Например,
```suggestion
		mlogger.Infof("failed to delete the payment card data: %w", err)
```